pkg/backend/display/diff_test.go

```diff
@@ -137,3 +137,63 @@ func TestDiffEvents(t *testing.T) {
 		})
 	}
 }
+
+func TestHasPolicyViolations(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name                   string
+		events                 []engine.PolicyViolationEventPayload
+		hasMandatoryViolations bool
+	}{
+		{
+			name:                   "no policy violations",
+			events:                 []engine.PolicyViolationEventPayload{},
+			hasMandatoryViolations: false,
+		},
+		{
+			name: "only advisory violations",
+			events: []engine.PolicyViolationEventPayload{
+				{
+					EnforcementLevel: "advisory",
+				},
+				{
+					EnforcementLevel: "advisory",
+				},
+			},
+			hasMandatoryViolations: false,
+		},
+		{
+			name: "has 1 mandatory violation",
+			events: []engine.PolicyViolationEventPayload{
+				{
+					EnforcementLevel: "mandatory",
+				},
+				{
+					EnforcementLevel: "advisory",
+				},
+			},
+			hasMandatoryViolations: true,
+		},
+		{
+			name: "has no mandatory violation",
+			events: []engine.PolicyViolationEventPayload{
+				{
+					EnforcementLevel: "disabled",
+				},
+				{
+					EnforcementLevel: "advisory",
+				},
+			},
+			hasMandatoryViolations: false,
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			hasMandatoryViolations := hasMandatoryPolicyViolations(tt.events)
+			assert.Equal(t, tt.hasMandatoryViolations, hasMandatoryViolations)
```

Comment:
nit:
These lines in particular are overly verbose.
Over 2/3 of the characters here are spent in saying the words "has", "mandatory", "violations" 4 times.

Versus:

```
got := hasMandatoryPolicyViolations(tt.give)
assert.Equal(t, tt.want, got)
```